src/server/connection.rs

```diff
@@ -1573,14 +1573,7 @@ impl Connection {
             #[cfg(target_os = "windows")]
             {
                 if !self.checked_multiple_session {
-                    let usid;
-                    match lr.option.user_session.parse::<u32>() {
-                        Ok(n) => usid = Some(n),
-                        Err(..) => usid = None,
-                    }
-                    if usid.is_some() {
-                        self.user_session_id = usid;
-                    }
+                    self.user_session_id = Some(crate::platform::get_current_process_session_id());
```

Comment:
![1707094949596](https://github.com/rustdesk/rustdesk/assets/14891774/457502e1-6093-4dbc-b653-63ed117c39bd)

Latest version on both sides, switching between console and rdp session for several times, cm shows up in wrong session.